src/NHibernate.Test/NHSpecificTest/NH3386/Fixture.cs

```diff
@@ -60,7 +60,7 @@ public void ShouldSupportNonRuntimeExtensionWithoutEntityReference()
 
 	public static class SqlServerFunction
 	{
-		[LinqExtensionMethod]
+		[LinqExtensionMethod(DBOnly = true)]
```

Comment:
Requiring users to add additional flag for existing methods is a breaking change.